linera-core/src/client/mod.rs

```diff
@@ -1123,7 +1129,7 @@ where
 
     /// Broadcasts certified blocks to validators.
     #[instrument(level = "trace", skip(committee, delivery))]
-    async fn communicate_chain_updates(
+    pub async fn communicate_chain_updates(
```

Comment:
If we add `pub` then they become part of the API contract and we no longer can change them.